src/type/definition.js

```diff
@@ -263,6 +317,10 @@ export class GraphQLScalarType {
     return parser ? parser(valueNode) : null;
   }
 
+  toJSON(): mixed {
```

Comment:
let's do toJSON in a separate PR. The right fix for that is probably to share the method rather than call through